lib/features/camera/camera_screen.dart

```diff
@@ -0,0 +1,110 @@
+                    ))),
+          ),
+          Positioned(
+              top: NartusDimens.padding40 + NartusDimens.padding4,
+              left: NartusDimens.padding16,
+              child: CircleButton(
+                size: NartusDimens.padding40,
+                iconPath: Assets.images.closeIcon,
+                semantic: S.current.close,
+                onPressed: () => context.pop(),
+              )),
+          Positioned(
+            left: 0, right: 0,
+            // bottom: NartusDimens.padding4 + NartusDimens.padding20,
+            bottom: MediaQuery.of(context).viewPadding.bottom,
+            child: Column(children: [
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  CircleButton(
+                    size: NartusDimens.padding40,
+                    iconPath: Assets.images.galleryIcon,
+                    semantic: S.current.openDeviceGallery,
+                    onPressed: () => context.goToHome(),
+                  ),
+                  Semantics(
+                    button: true,
+                    enabled: true,
+                    excludeSemantics: true,
+                    explicitChildNodes: false,
+                    label: S.current.captureMediaButton,
+                    child: Container(
+                      width: NartusDimens.padding40 +
+                          NartusDimens.padding32 +
+                          NartusDimens.padding4,
+                      height: NartusDimens.padding40 +
+                          NartusDimens.padding32 +
+                          NartusDimens.padding4,
+                      decoration: BoxDecoration(
+                          shape: BoxShape.circle,
+                          border: Border.all(
+                              color: NartusColor.white,
+                              width: NartusDimens.padding4),
+                          color: Colors.transparent),
+                      padding: const EdgeInsets.all(NartusDimens.padding2),
+                      child: Container(
+                        decoration: BoxDecoration(
+                            shape: BoxShape.circle,
+                            border:
+                                Border.all(color: NartusColor.white, width: 4),
+                            color: Colors.white),
+                        child: NartusButton.text(
+                          label: '',
+                          onPressed: () => context.gotoPreviewMediaScreen(),
+                        ),
+                      ),
+                    ),
+                  ),
+                  CircleButton(
+                    size: NartusDimens.padding40,
+                    iconPath: Assets.images.flipIcon,
+                    semantic: S.current.flipCamera,
+                    onPressed: () => context.goToHome(),
```

Comment:
why do we go to home here? if there's no action yet, just leave empty action
also need semantic here